### DISCUSSION TITLE
role based redirect on login

### DISCUSSION THREAD

**Discussion by dimdam** [filamentphp/filament#1678] [Help]
Is it possible via filament to do this?

User fills credentials and if he is not admin,then redirect him to another  authenticated page

**Comment by studiowizjo** [comment]
1. Create own login class extending **Filament\Http\Livewire\Auth\Login::class** 
2. Overwite in it **authenticate()** method
3. In config/filament.php point **auth.pages.login** key to your class

**Comment by dimdam** [comment]
Thank you for your answer. I thought of using Laravel breeze in order to redirect users based on their roles and only if they're admins can use filament. I don't know if this makes any sense?

**Reply by studiowizjo** [reply]
It is up to you. As a rule, IMHO the Filament logging system is designed as a quick, starting point, so if any major modifications are needed, you can use any other package for authentication purposes. However, when it comes to this, it seems to me that the solution described above will work. 

You can take different approach, for example:

1. Create middleware and there put redirection logic
2. You can also make a redirection on the dashboard resource page. To do this create method [viewAny()](https://filamentphp.com/docs/2.x/admin/resources#authorization).

**Reply by ehs4nm** [reply]
@dimdam 
Try this package:[ jeffgreco13/filament-breezy](https://github.com/jeffgreco13/filament-breezy/)

**Reply by dimdam** [reply]
I'm still not sure but maybe I'll try implementing policies because as @studiowizjo mentioned , filament is designed as a quick starting point,so there's no need in my case to modify it or extend it with other packages. Thanks for the input though.

**Reply by hellohendri** [reply]
> 

Is it possible to add this package to my live filament project?
